build/gen-windows-versioninfo.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Run goversioninfo to generate the resource.syso to embed version info.
+set -eux
+
+NAME="One Policy Agent (OPA)"
```

Comment:
```suggestion
NAME="Open Policy Agent (OPA)"
```
Small typo! :sweat_smile:

Reply:
haha pretty important typo

Reply:
How does it go again?... One Policy Agent to rule them all, One Policy Agent to find them, One Policy Agent to bring them all, and in the darkness bind them with Rego.

Reply:
haha I need that on a shirt, with Sauron wearing the OPA logo as a helmet

But they were, all of them, deceived, for another policy agent was made.